aries_vcx_core/src/anoncreds/credx_anoncreds.rs

```diff
@@ -383,6 +384,7 @@ impl BaseAnonCreds for IndyCredxAnonCreds {
         let cred_request = serde_json::from_str(cred_req_json)?;
         let cred_values = serde_json::from_str(cred_values_json)?;
 
+        // TODO: Might need to qualify with offer method or something - look into how vdrtools does it
```

Comment:
It still is a TODO, yeah. I left it because I noticed this now and I don't really know how to approach this whole qualification thing. I was thinking of tackling this in a separate PR, maybe directly in the `anoncreds-rs` implementation.